### ISSUE TITLE
root user has all privileges except for grant_priv

### ISSUE THREAD

**Issue by aaw** [aptible/docker-mysql#10]
We may just need to explicitly update the `mysql.user` table during `--initialize` as [this stackoverflow issue](http://dba.stackexchange.com/a/16410) suggests.

**Comment by shreeve** [comment]
This "grant option" might also be helpful.

https://dev.mysql.com/doc/refman/5.1/en/privileges-provided.html#priv_grant-option

```
https://github.com/aptible/docker-mysql/blob/53ee577664895292b0ef344c8838436836d5a4c8/5.6/run-database.sh#L13-L14
```

**Comment by krallin** [comment]
Fixed in #15
